### ISSUE TITLE
sits_plot output of sits_classify with more than one band

### ISSUE THREAD

**Issue by pedro-andrade-inpe** [e-sensing/sits#96]
When calling `sits_plot` with the output of `sits_classify` using more than one band, in only draws the first band as default. Shouldn´t it draw all them? In the example below, it shows only ndvi.

```R

data(point_MT_6bands)
data(samples_MT_9classes)

samples.tb <- sits_select(samples_MT_9classes,
                          bands = c("ndvi", "evi"))
model <- sits_train(samples.tb, sits_svm())
point.tb <- sits_select(point_MT_6bands, bands = c("ndvi", "evi"))

result <- sits_classify(point.tb, model)

sits_plot(result)
```

**Comment by gilbertocamara** [comment]
This is not easy to do. The sits_plot_classification function is adapted based on a function done by Victor Maus for the "dtwSat" package, which was not simple to understand. I will try to include more bands, but it is not obvious how to do.

**Comment by gilbertocamara** [comment]
Problem solved in version 0.9.2
